x-pack/plugins/apm/server/lib/service_map/get_service_anomalies.ts

```diff
@@ -62,7 +62,13 @@ export async function getServiceAnomalies(
     return [];
   }
 
-  const { jobs: apmMlJobs } = await ml.anomalyDetectors.jobs('apm');
+  let apmMlJobs: AnomalyDetectionJob[];
+  try {
+    const { jobs } = await ml.anomalyDetectors.jobs(APM_ML_JOB_GROUP_NAME);
+    apmMlJobs = jobs;
+  } catch (error) {
+    return [];
```

Comment:
for readability's sake, should we set apmMlJobs to `[]` and return (outside of the try/catch) if `apmMlJobs.length === 0`?

Reply:
I must admit, I found it more readable to return from the catch but either works.